pipemgr/src/pipemgr/concourse/routes.py

```diff
@@ -0,0 +1,11 @@
+PIPELINE_CONFIG = "/api/v1/teams/{team_name}/pipelines/{pipeline_name}/config"
```

Comment:
You might want to use a function or method to build the URLs instead of keeping all these magic constants around. You could add this to your custom requests session. I do something like that [here](https://github.com/openstax/ghzh-clients/blob/master/ghzh/zenhub/client.py#L52-L70)

Reply:
Yes, I had considered that. This routes file is not used currently, it is just an idea that I forgot to remove. 

Additionally, I should probably move `get_pipeline` and `set_pipeline` from the pipemgr specific concourse interface (currently named `ConcourseHandler`) into a more generic concourse interface since these methods can be reused. In fact, I think `ConcourseHandler` could be moved into the concourse package and renamed to `ConcourseService`.